### PR TITLE
[Ready for Review]:small changes and Demo UI fixed

### PR DESCRIPTION
**Pull request by FarrukhSubhaniGH**
1. Overflow hidden added on accordion content.
2. Button height and min-width adjusted.
3. Accordion button space variable added
4. Demo Page UI Fixed

### CODE REVIEW

File: libs/static/src/scss/components/_accordion.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 .fds-btncollapse,
 .fds-btnaccordion {
```
Comment (zeenainali) [inline libs/static/src/scss/components/_accordion.scss:6]: @FarrukhSubhaniGH  These classes should be as per the FDS classes.

File: libs/static/src/scss/tools/tool.grid.scss [hunk 2]
```diff
@@ -62,7 +62,8 @@
     margin-left: 0;
 
     > .col,
-    > [class*='col-'] {
+    > [class*='col-'],
```
Comment (zeenainali) [inline libs/static/src/scss/tools/tool.grid.scss:65]: @FarrukhSubhaniGH  Removes this as we are not using with single dash.

File: libs/static/src/views/components/_partial.colorplatte.html [hunk 3]
```diff
@@ -8,38 +8,38 @@ <h3 class="h3">Color Palette</h3>
...
             <div class="col colors__infoLeft">bg class:</div>
-            <div class="col-5 colors__infoRight bg--primary">bg--primary</div>
+            <div class="col--5 colors__infoRight bg--primary">bg--primary</div>
           </li>
         </ul>
       </div>
 
-      <div class="col-sm-6 col-md-3">
+      <div class="col--sm--6 col--md--3">
         <div class="my--2 text--body-lg text--bold color-name-title">
```
Comment (zeenainali) [inline libs/static/src/views/components/_partial.colorplatte.html:35]: color-name-title is this class for FDS or for project and if for FDS then should be as per the other classes.

File: libs/static/src/scss/components/_accordion.scss (hunk 1 above)
Comment (FarrukhSubhaniGH) [reply libs/static/src/scss/components/_accordion.scss:6]: fixed

File: libs/static/src/scss/tools/tool.grid.scss (hunk 2 above)
Comment (FarrukhSubhaniGH) [reply libs/static/src/scss/tools/tool.grid.scss]: fixed

File: libs/static/src/views/components/_partial.accordion.html [hunk 4]
```diff
@@ -4,7 +4,7 @@ <h2>Collapse</h2>
 
     <div class="jumbotron">
```
[the remark this answers is not in the document]
Comment (FarrukhSubhaniGH) [reply libs/static/src/views/components/_partial.accordion.html:5]: @zeenainali we can leave it as it is the reason is it is only for the demo page

File: libs/static/src/views/components/_partial.colorplatte.html (hunk 3 above)
Comment (FarrukhSubhaniGH) [reply libs/static/src/views/components/_partial.colorplatte.html:35]: Fixed

**Pull request merged**